libraries_skip.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+    <!--Contains the Complete Names for each library test to be skipped.-->
+    <ItemGroup>
+        <!--SkipTests Include="Microsoft.CSharp.RuntimeBinder.Tests.DynamicDebuggerProxyTests.Items_Empty" /-->
```

Comment:
Does this means that we are going to loose the ability to disable tests per arch, runtime, target platforms, etc? If not how is that going to be automated?

Reply:
Shouldn't you be able to use a `Condition` for that? This is just an MSBuild props file.

Reply:
There is a lot of detection that happens at runtime rather than compile time; so I just want to make sure we don't loose that.

Also, if we use the `Condition` for that the automation would be sensitive to our MSBuild props and being able to identify if an item group exists with a condition that meets that, etc. 

Maybe a smarter way would be to make `SkipTests` have custom metadata, say `Platform=Linux`, `Arch=x64`, `Runtime=Mono`, of course if no `Runtime` is specified, it would be disabled on both, same for platform, arch, etc. Then when we generate the command to disable the tests we can parse that metadata and see if it applies for the current target configuration.